sdk/docs/manually-written/sdk/sdlc-howtos/smart-contracts/build/index.rst

```diff
@@ -99,7 +98,7 @@ The location where the .dar is created can be overridden using the ``--output``
   build-options:
     - --output=./output-bin/my-package.dar
 
-See :ref:`Daml Build Options <daml-build-flags>` for a full list of daml build options, or run ``daml build --help``, which includes options for changing the :brokenref:`LF <lf-version>` version and configuring warnings. All of these options can also be provided via ``build-options`` above.
```

Comment:
In the 3.x line, there is currently only Daml-LF 2.1.

This text could be reverted if/when we return to a world where people really do have to specify Daml-LF versions, but right now it's just noise.